### DISCUSSION TITLE
最最近有个绳匠升级速度跟开了一样

### DISCUSSION THREAD

**Discussion by morning134** [share121/inter-knot#166] [其他]
最近有个绳匠，一个多月的时间升了50多级，这速度跟开了一样，ta到底是怎么做到的？我听别人说ta什么委托都接，小到帮情侣过周年，大到各种极难战斗，来者不拒。
       难道真是法厄同转世重修了？

**Comment by DeepslateQAQ** [comment]
```
哥们别用代码块说话
```

**Comment by Mu-7593** [comment]
不知道是不是有什么特殊消息渠道，每次真正高报酬的委托都被那个绳匠接走，之后出现的同类型委托又大多是骗子，等到人人都觉得不会再有了的时候又让他接到真的了，好气人 💢

**Comment by reversed-PAI** [comment]
话说法厄同好久没有出现了，不会是账号被黑了吧🤔

**Comment by Liangjl-2024** [comment]
笑死，大名鼎鼎的法厄同会被区区黑客黑掉账号？要真是这样的话我学邦布嗯呢一百声！
